.github/ISSUE_TEMPLATE/feedback.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Not entirely sure if GH issues is the best way for collecting feedback.

I’d rather enable Discussions section in repo and move such things there leaving issues for more actionable things

Reply:
I think you are right. I got this idea from the github CLI, but looking at there feedback section, it's not super actionable. Removed.